### ISSUE TITLE
Feature request / feedback: filters and workflow keyboard shortcuts 

### ISSUE THREAD

**Issue by spaceman-cb** [Oxygem/Kanmail#209]
Hi Kanmail team,

### TLDR:
This app is almost **_exactly_** what I've been looking for, but not quite. A few keyboard shortcuts & filters would make it perfect

### Background
I found your app while looking for an email client to help me track the status of different conversations across multiple inboxes/folders. In the past, I have tried using labels/folders for this (e.g. 'to do', 'doing', 'done'), but neither gmail nor any other mail clients I can find provide a good UI or a good set of shortcuts for this.

This is exactly what I have been looking for several years; however, without the following conveniences, it is not quite yet to the point where I could migrate to it.

#### Keyboard shorcuts: move thread to prev/next/nth column
What I want to do is to be able to quickly flip through and preview my mail threads, and then move them forward/backward in my workflow/columns with a single button press. something like Gmail's <kbd> [</kbd> / <kbd> ]</kbd> shortcuts, except instead of archiving the thread, it would move it to the previous or next column, and then show me the next thread in the inbox/currently selected column.

An alternative idea would be using the number keys(?) to send the current thread to col 1-9 respectively?

#### Option to make last column = archive (?)
Just a thought, but often once I'm done with an email, I want to banish it to the archive.

### Filter columns by current label
As far as I can tell, selecting a label/folder changes the contents of the leftmost pane, but doesn't filter the displayed columns.
If I am working on a set of emails in one folder, it would be nice if I could show ONLY those emails on the kanban board to cut down on clutter and help me focus on the work at hand.

---

This is a very exciting app, and I hope to see more amazing features soon!
Thanks!

**Comment by Fizzadar** [comment]
Hey @spaceman-cb! Thank you for the feedback, this is super helpful! Will address each below:

**Keyboard shortcuts**

This is supported already - using the arrow keys! But this is also poorly documented (see: https://kanmail.io/docs/keyoard-shortcuts). Issue #3 tracks making the shortcuts customizable.

This should allow you to jump left/right/up/down between threads and columns as desired, although do let me know of any more shortcuts that would be useful!

**Last column = archive**

If I understand this you mean to have the final column on the right be archive? You can add this as a column yourself to achieve that currently (and "standard" folders can be columns), does that solve this for you?

**Filter columns by label**

You are correct currently the left panel only affects the left, "main", column. I really like the idea of filtering by the main column however. I actually think some work has been done before on this but will confirm that!
